pkg/kubectl/resource_printer.go

```diff
@@ -413,6 +413,7 @@ var podTemplateColumns = []string{"TEMPLATE", "CONTAINER(S)", "IMAGE(S)", "PODLA
 var replicationControllerColumns = []string{"NAME", "DESIRED", "CURRENT", "AGE"}
 var replicaSetColumns = []string{"NAME", "DESIRED", "CURRENT", "AGE"}
 var jobColumns = []string{"NAME", "DESIRED", "SUCCESSFUL", "AGE"}
+var scheduledJobColumns = []string{"NAME", "SCHEDULE", "SUSPEND", "ACTIVE", "LASTSCHEDULE"}
```

Comment:
Change `LASTSCHEDULE` to `LAST-SCHEDULE`. By convention we use `-` as the word separator, see https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md#output-conventions